be/src/exec/vectorized/analytor.cpp

```diff
@@ -379,17 +379,19 @@ void Analytor::get_window_function_result(int32_t start, int32_t end) {
     }
 }
 
-bool Analytor::is_partition_finished() {
+bool Analytor::is_partition_finished(int64_t found_partition_end) {
```

Comment:
Use `const int64_t&` may avoid copying? (I have no idea whether it is necessary.)

Reply:
`const int64_t&` copies a pointer, which is also 64 bits. And passing by value often increases cache locality, because passing by reference must get the memory address of this variable.